src/plugins/unified_search/public/query_string_input/query_string_input.test.tsx

```diff
@@ -77,6 +77,15 @@ function wrapQueryStringInputInContext(testProps: any, storage?: any) {
   const defaultOptions = {
     screenTitle: 'Another Screen',
     intl: null as any,
+    deps: {
+      unifiedSearch: unifiedSearchPluginMock.createStartContract(),
+      data: dataPluginMock.createStartContract(),
+      appName: testProps.appName || 'test',
```

Comment:
just interesting how it worked before without 'test' ?  Maybe appName is an optional? And we don't need to hardcode that value as you do in other files?